edgelet/contrib/config/linux/template.toml

```diff
@@ -387,6 +387,24 @@
 # [edge_ca]
 # auto_generated_edge_ca_expiry_days = 90
 
+# ==============================================================================
+# Edge CA certificate auto-renewal
+# ==============================================================================
+#
+# Manage auto-renewal of the Edge CA certificate. Generally, this applies when the
+# Edge CA is configured as Quickstart or when the Edge CA has an issuance `method`
+# set. Edge CA certs loaded from files generally cannot be auto-renewed as the Edge
+# runtime will not have enough information to renew them.
```

Comment:
In case of Edge CA loaded from file, I suppose we will log a warning that we cannot renew cert and continue operations, right?

Reply:
No, I left out the setting for preloaded certs. It will have no effect if set.